src/lib/api/rest.ts

```diff
@@ -43,6 +43,6 @@ export async function put<T>(path: string, data: any = {}) {
   return response.body;
 }
 
-export async function del<T>(path: string) {
-  return await Request.delete<T>(apiUrl(path)).withCredentials();
+export function del<T>(path: string) {
```

Comment:
Is there any reason why we removed the `async` `await` from this method?

Reply:
Because the property 'send' does not exist on type `Promise<any> `

> `del<any>(`/chatChannels/${channelId}/message`).send({ message: { id: messageId } });`